src/sentry/static/sentry/app/actionCreators/organization.tsx

```diff
@@ -9,6 +9,54 @@ import TeamActions from 'app/actions/teamActions';
 import {Client} from 'app/api';
 import ProjectsStore from 'app/stores/projectsStore';
 import TeamStore from 'app/stores/teamStore';
+import {Organization} from 'app/types';
+
+async function fetchOrg(
+  api: Client,
+  slug: string,
+  detailed: boolean
+): Promise<Organization> {
+  const org = await api.requestPromise(`/organizations/${slug}/`, {
+    query: {detailed: detailed ? 1 : 0},
+  });
+
+  if (!org) {
+    throw new Error('retrieved organization is falsey');
+  }
+
+  OrganizationActions.update(org, {replace: true});
+  setActiveOrganization(org);
+
+  // eslint-disable-next-line consistent-return
```

Comment:
Isn't the return of this function consistent though?

Reply:
@markstory my bad at one point the throw a few lines up was a `return;` and it was marked as inconsistent